code/R3/R3.Core/Geometry/Vector3D.cs

```diff
@@ -133,10 +133,6 @@ public int GetHashCode( double tolerance )
 
 		public bool Compare( Vector3D other, double threshold )
 		{
-			// NOTE: This is here because when the vector is infinite, it fails the tolerance checks below.
-			if( ( X == other.X ) && ( Y == other.Y ) && ( Z == other.Z ) && W == other.W )
```

Comment:
As mentioned on the other thread, I'd prefer to leave this check in and am wondering what is your motivation for removing it?